src/uu/install/src/install.rs

```diff
@@ -283,7 +287,7 @@ pub fn uu_app() -> Command {
             Arg::new(OPT_NO_TARGET_DIRECTORY)
                 .short('T')
                 .long(OPT_NO_TARGET_DIRECTORY)
-                .help("(unimplemented) treat DEST as a normal file")
+                .help("treat DEST as a normal file")
```

Comment:
The TODO on line 286 is no longer necessary as you implement `--no-target-directory`